src/sentry/eventstore/base.py

```diff
@@ -0,0 +1,82 @@
+from __future__ import absolute_import
+
+from sentry import nodestore
+from sentry.utils.services import Service
+
+
+class EventStorage(Service):
```

Comment:
deep philosophical question that I am sure will prevent us all from sleeping:
EventStore or EventStorage ?

Reply:
EventStore matches the module name.

Reply:
As far as precedent goes, `filestore`, `nodestore` and `tagstore` all use the `-Storage` suffix for their implementation classes.

Reply:
Yeah, i based this off the other services naming convention